src/openforms/js/compiled-lang/nl.json

```
@@ -1008,7 +1008,7 @@
   "tjRtQk": [
     {
       "type": 0,
-      "value": "Submission confirmation email template"
+      "value": "Bevestings e-mailsjabloon"
```

```suggestion
      "value": "Bevestigingsmailsjabloon"
```
[quick google](https://www.villamedia.nl/artikel/taaltip-bevestigings-e-mail)